node_modules/col-core/lib/email.js

```diff
@@ -52,6 +52,26 @@ var EVENT_NAMES = module.exports.EVENT_NAMES = {
   'EMAIL_SENT': 'emailSent'
 };
 
+// Variable that maps the original image paths to their hashed counter parts (if any)
+var imageManifest = {};
+
+// Cache the image manifest
+var manifestPath = path.resolve(__dirname, '..', '..', '..', config.get('apache.documentRoot'), 'images-rev-manifest.json');
```

Comment:
Yes, that's what `documentRoot` is supposed to do. This is supposed to be `path.resolve(config.get('apache.documentRoot'), 'images-rev-manifest.json')` but I forgot to clean up my debugging code :(